From 59392e29ca0d864d26c814e28034085be3bc2c9d Mon Sep 17 00:00:00 2001
From: Angela Stegmaier <angelabaker@ti.com>
Date: Mon, 8 Mar 2021 19:03:59 +0530
Subject: [PATCH 6/9] drivers: media: platorm: Kconfig: Add Video encoder
 kconfig entries

Add video encoder basic config to Kconfig
and select the required V4L2 modules.

Signed-off-by: Angela Stegmaier <angelabaker@ti.com>
Reviewed-by: Benoit Parrot <bparrot@ti.com>
Signed-off-by: Sidraya Jayagond <sidraya.bj@ti.com>
---
 drivers/media/platform/Kconfig | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/drivers/media/platform/Kconfig b/drivers/media/platform/Kconfig
index 49c9f0a889a3..7e4563a540f3 100644
--- a/drivers/media/platform/Kconfig
+++ b/drivers/media/platform/Kconfig
@@ -515,6 +515,19 @@ config VIDEO_IMG_VXD_DEC
 	    Imagination D5520 (Video Decoder) hardware.
 	    The module name when built is vxd-dec.
 
+config VIDEO_IMG_VXE_ENC
+        tristate "IMG VXE ENC (Video Encoder) driver"
+        depends on VIDEO_DEV && VIDEO_V4L2
+        depends on ARCH_K3_J721E_SOC || COMPILE_TEST
+        select VIDEOBUF2_CORE
+        select VIDEOBUF2_DMA_CONTIG
+        select VIDEOBUF2_DMA_SG
+        select V4L2_MEM2MEM_DEV
+        help
+          This is an IMG VXE ENC V4L2 driver that adds support for the
+          Imagination VXE384 (Video Encoder) hardware.
+          The module name when built is vxe-enc.
+
 config VIDEO_QCOM_VENUS
 	tristate "Qualcomm Venus V4L2 encoder/decoder driver"
 	depends on VIDEO_DEV && VIDEO_V4L2
-- 
2.17.1

